webui/react/src/components/ExperimentActionDropdown.tsx

```diff
@@ -97,6 +104,25 @@ const ExperimentActionDropdown: React.FC<Props> = ({
           openCommand(tensorboard);
           break;
         }
+        case Action.SwitchPin: {
```

Comment:
fix: GitHub won't let me put this comment there, but at line 194, you should check whether

1. `action === Action.SwitchPin`
and if so,
2. `label: pinSet.has(id) ? 'Unpin' : 'Pin'`